resources/views/livewire/slack-settings-form.blade.php

```diff
@@ -0,0 +1,104 @@
+
+
+    <div class="col-md-12" style="border-top: 0px;">
+        @if (session()->has('save'))
+        <div class="alert alert-success">
+            {{session('save')}}
+        </div>
+        @endif
+            @if(session()->has('success'))
+                <div class="alert alert-success">
+                    {{session('success')}}
+                </div>
+            @endif
+            @if(session()->has('error'))
+                <div class="alert alert-danger">
+                    {{session('error')}}
+                </div>
+            @endif
+            @if(session()->has('message'))
+                <div class="alert alert-danger">
+                    {{session('message')}}
+                </div>
+            @endif
+        <form wire:submit.prevent ="submit">
+        {{csrf_field()}}
+
+        <!--slack endpoint-->
+        <div class="form-group required {{ $errors->has('slack_endpoint') ? 'error' : '' }}">
+            <div class="col-md-2">
+                {{ Form::label('slack_endpoint', trans('admin/settings/general.slack_endpoint')) }}
+            </div>
+            <div class="col-md-10">
+                @if (config('app.lock_passwords')===true)
+{{--                                    {{ Form::text('slack_endpoint', old('slack_endpoint', $setting->slack_endpoint), array('class' => 'form-control','disabled'=>'disabled','placeholder' => 'https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX', 'id' => 'slack_endpoint')) }}--}}
+                    <p class="text-warning"><i class="fas fa-lock"></i> {{ trans('general.feature_disabled') }}</p>
+                    <input type="text" wire:model.lazy="slack_endpoint" id="slack_endpoint" class= 'form-control' placeholder="'https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX'" {{old('slack_endpoint', $slack_endpoint)}} ><br>
+                @else
+                    <input type="text" wire:model.lazy="slack_endpoint" id="slack_endpoint" class= 'form-control' placeholder="'https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX'" {{old('slack_endpoint', $slack_endpoint)}} ><br>
```

Comment:
I think we can remove the extra `'`s in the placeholder so the user doesn't think they have to surround their endpoint in quotes

![CleanShot 2023-02-02 at 15 54 29](https://user-images.githubusercontent.com/1141514/216477537-09776494-8dbc-47df-8a4a-a789022081bb.png)

```suggestion
                    <input type="text" wire:model.lazy="slack_endpoint" id="slack_endpoint" class= 'form-control' placeholder="https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX" {{old('slack_endpoint', $slack_endpoint)}} ><br>
                @else
                    <input type="text" wire:model.lazy="slack_endpoint" id="slack_endpoint" class= 'form-control' placeholder="https://hooks.slack.com/services/XXXXXXXXXXXXXXXXXXXXX" {{old('slack_endpoint', $slack_endpoint)}} ><br>
```